wpilib/wpilib/sendable.py

```diff
@@ -54,20 +57,11 @@ def setSubsystem(self, subsystem):
         """
         raise NotImplementedError
 
-    def initSendable(self, builder):
+    def initSendable(self, builder: SendableBuilder) -> None:
         """
         Initializes this Sendable object.
 
         :param builder: sendable builder
         :type builder: :class:`wpilib.SendableBuilder`
```

Comment:
So this doesn't actually link right now. Given you're adding a type hint, perhaps remove this redundant line.